HW13/proxy_txt_reader.py

```diff
@@ -0,0 +1,39 @@
+from HW13.txt_reader import TxtReader
+from HW13.abstract_reader import Reader
+from HW13.abstract_writer import Writer
+
+
+class TxtProxyWriterReader(Reader, Writer):
+    def __init__(self, file_path):
+        self.__result = ''
+        self.__is_actual = False
+        self.__txt_reader = TxtReader(file_path)
+
+    def read_file(self):
+        if self.__is_actual:
+            return self.__result
+        else:
+            self.__result = self.__txt_reader.read_file()
+            self.__is_actual = True
+            return self.__result
+
+    def write_file(self, text):
+        with open(self.__txt_reader.file_path, 'a') as file:
+            file.write('\n' + text)
+        self.__result = text
+
+    def overwrite_file(self, text):
+        with open(self.__txt_reader.file_path, 'w') as file:
+            file.write(text)
+        self.__result = text
```

Comment:
And where is proxy? Think about moments where you can just do nthg. 
Also do not use with open here...it is for higher layer